config/initializers/controllers.rb

```diff
@@ -38,31 +38,24 @@ def contracts_not_required
   end
 
   def registration
-    user = (request.format == :json) ? current_human_user : current_user
```

Comment:
`current_human_user` is not a method available in this class (only available in descendents of the api controller from `openstax_api`)